node/network/collator-protocol/src/collator_side/mod.rs

```diff
@@ -986,8 +1021,10 @@ where
 				FromOverseer::Signal(BlockFinalized(..)) => {}
 				FromOverseer::Signal(Conclude) => return Ok(()),
 			},
-			relay_parent = state.active_collation_fetches.select_next_some() => {
+			res = state.active_collation_fetches.select_next_some() => {
+				let (relay_parent, peer_id) = res;
```

Comment:
```suggestion
			(relay_parent, peer_id) = state.active_collation_fetches.select_next_some() => {
```

Reply:
Weird, that did not compile previously for some reason, but does now. Thanks.